operations/observability/mixins/cross-teams/dashboards/gitpod-overview.libsonnet

```diff
@@ -62,54 +62,26 @@ local runningWorkspacesGraph =
   .addSeriesOverride({ alias: 'Regular Not Active', color: '#FADE2A' })
 ;
 
-local noisyNeighborGraph =
+local wsNodeLoadAverageGraph =
   graphPanel.new(
-    '$cluster: # of Regular Workspaces under Noisy Neighborhood',
+    datasource='$datasource',
+    title="$cluster: Workspace node's normalized load average",
     description=
     |||
-      When a Node has a normalized load average higher than 1, that means that the workloads are demanding more compute power than the node is able to provide.
-      When that happens, then all workloads may have degraded performance.
-
-      This panel do not answer the question: "What is workload is demanding so much CPU?"
-      But it does answer: "How many regular workspaces are experiencing degraded performance because of the noisy neightbor effect?"
+      Nodes with a high normalized load average do not represent a real problem, it only means that pods should probably not be scheduled to them.
 
-      To find out the noisy neighbor, drill down to '%(dashboardNamePrefix)sNodes Overview' and explore nodes that have normalized load average higher than 1.
-    ||| % _config,
-    datasource='$datasource',
+      If you'd like to see more details about resource consumption of a particular node, you can do so by clicking at the node name.
+    |||,
     format='none',
-    stack=false,
     fill=1,
     fillGradient=5,
     min=0,
     repeat='cluster',
-    nullPointMode='null as zero'
   )
-  .addLink(link.dashboards(
-    title='%sNodes Overview' % _config.dashboardNamePrefix,
-    tags=[''],
-    url='d/gitpod-nodes-overview/gitpod-nodes-overview',
-    keepTime=true,
-    targetBlank=true,
-    includeVars=true,
-  ))
   .addTarget(prometheus.target(
     |||
-      sum(
-        count(
-          sum(container_cpu_cfs_periods_total{%(clusterLabel)s=~"$cluster", container="workspace"}) by (pod, node, %(clusterLabel)s) # Sum just to aggregate all containers into a single pod
-          *
-          on(pod) group_left() kube_pod_labels{component="workspace", workspace_type="regular", %(clusterLabel)s=~"$cluster"} # join to make sure we only count pods of regular workspaces
-        ) by (node, %(clusterLabel)s)
-
-        and # We are correlating workspaces with Nodes with high normalized load average
-
-        (
-          sum(
-            instance:node_load1_per_cpu:ratio{cluster=~"$cluster"}
-          ) by (node, %(clusterLabel)s) # Sum to remove all labels except node and %(clusterLabel)s, labels must match with the first part of the query
-        ) > 1
-      ) by (%(clusterLabel)s)
-    ||| % _config, legendFormat='# of workspaces'
+      topk(5, sum(nodepool:node_load1:normalized{%(clusterLabel)s=~"$cluster", nodepool=~".*workspace.*"}) by (node))
```

Comment:
Loks like Victoriametrics [startet supporting topk](https://docs.victoriametrics.com/MetricsQL.html#topk). This is awesome.

Reply:
topk has always worked, the [@ modifier ](https://prometheus.io/docs/prometheus/latest/querying/basics/#modifier)that doesn't :)